flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTableDropConstraint.java

```diff
@@ -38,8 +38,11 @@
      * @param pos Parser position
      */
     public SqlAlterTableDropConstraint(
-            SqlIdentifier tableID, SqlIdentifier constraintName, SqlParserPos pos) {
-        super(pos, tableID);
+            boolean ifExists,
```

Comment:
Maybe change to `SqlAlterTableDropConstraint(SqlParserPos, SqlIdentifier, SqlIdentifier, boolean)`?